ext/ffi/ffi.c

```diff
@@ -511,6 +511,10 @@ static zend_always_inline void zend_ffi_cdata_to_zval(zend_ffi_cdata *cdata, voi
 				kind = type->enumeration.kind;
 				goto again;
 			case ZEND_FFI_TYPE_POINTER:
+				if (debug_union) {
+					ZVAL_LONG(rv, (uintptr_t)*(void**)ptr);
+					return;
+				}
```

Comment:
It should be okay to move this below the null pointer check below, so that one gets printed as PHP null.

Reply:
Makes sense. Thanks!